src/libraries/System.Runtime.InteropServices/tests/ComInterfaceGenerator.Tests/ImplicitThisTests.cs

```diff
@@ -60,7 +60,7 @@ public NativeObject(void* pointer)
                 public VirtualMethodTableInfo GetVirtualMethodTableInfoForKey(Type type)
                 {
                     Assert.Equal(typeof(INativeObject), type);
-                    return new VirtualMethodTableInfo((IntPtr)_pointer, new ReadOnlySpan<IntPtr>(*(void**)_pointer, 2));
+                    return new VirtualMethodTableInfo(_pointer, (void**)_pointer);
```

Comment:
I think you are missing the dereference to get to the vtable.

Reply:
What's the best notation for that? I changed it to `*(void***)_pointer`, but not sure if that's the ideal cast/dereference order.